src/layouts/Header.tsx

```diff
@@ -16,7 +16,13 @@ const Header = () => {
         <link rel="manifest" href="/site.webmanifest" />
         <title>Welcome CourtCanva</title>
       </Head>
-      <NextHeadSeo title="CourtCanva" canonical="" description="" customMetaTags={[]} og={{}} />
+      <NextHeadSeo
+        title="CourtCanva Admin"
+        canonical=""
+        description=""
+        customMetaTags={[]}
+        og={{}}
+      />
     </>
```

Comment:
do we need SEO for CMS?

Reply:
No, we do not need SEO. I guess it's from the old cc-app project starter.